src/components/PeopleTable.tsx

```diff
@@ -1,5 +1,113 @@
+import { useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../types';
+import { PersonLink } from '../types/PersonLink';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  people: Person[];
+};
+
+function getVisiblePeople(
+  people: Person[],
+  sex: string,
+  query: string,
+  centuries: string[],
+  sort: string,
+  order: string,
+) {
+  let copyPeople = [...people];
+
+  if (sex) {
+    copyPeople = copyPeople.filter(person => person.sex === sex);
+  }
+
+  if (query) {
+    copyPeople = copyPeople.filter(
+      person =>
+        person.name.toLowerCase().includes(query) ||
+        person.motherName?.toLowerCase().includes(query) ||
+        person.fatherName?.toLowerCase().includes(query),
+    );
+  }
+
+  if (centuries.length !== 0) {
+    copyPeople = copyPeople.filter(person =>
+      centuries.includes(`${person.born + 100}`.slice(0, 2)),
+    );
+  }
+
+  if (sort) {
+    switch (sort) {
+      case 'name':
+        copyPeople.sort((el1, el2) => el1.name.localeCompare(el2.name));
+        break;
+      case 'sex':
+        copyPeople.sort((el1, el2) => el1.sex.localeCompare(el2.sex));
+        break;
+      case 'born':
+        copyPeople.sort((el1, el2) => el1.born - el2.born);
+        break;
+      case 'died':
+        copyPeople.sort((el1, el2) => el1.died - el2.died);
+        break;
+    }
+  }
+
+  if (order) {
+    return copyPeople.reverse();
```

Comment:
The `order` variable is expected to be a string that determines the sort order. However, the function `sortClick` uses 'desk' as a value for descending order, which seems to be a typo. It should likely be 'desc' instead of 'desk'.